marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserArtifactsCommand.java

```diff
@@ -100,7 +100,7 @@ private PropertiesModuleManager getModulesManager() {
         }
         //Initializing the 'PropertiesModuleManager' instance so that timestamp file (local-user-modules-deploy-timestamps.properties)
         // is read and only the modified files are deployed to server
-        pmm.initialize();
+        //pmm.initialize();
```

Comment:
I recommend deleting this line and the comments above it - keeping it will otherwise be confusing.